.travis.yml

```diff
@@ -12,9 +12,5 @@ env:
   - SCENERIO=in
   - SCENERIO=notify
   - SCENERIO=out
-install: pip install ansible molecule==2.22rc6 docker-py pre-commit pylint pep257
-before_install:
-  - cd ..
-  - ln -s ansible-pacifica pacifica
-  - cd pacifica
+install: pip install ansible molecule==2.22 docker-py pre-commit pylint pep257
```

Comment:
At this point the molecule version dependency isn't really required anymore. I think you can just install molecule without any version constraints.

Reply:
fixed